front/src/modules/settings/data-model/object-details/components/SettingsObjectItemTableRow.tsx

```diff
@@ -26,7 +26,11 @@ const StyledNameTableCell = styled(TableCell)`
 
 const StyledTag = styled(Tag)`
   box-sizing: border-box;
-  height: ${({ theme }) => theme.spacing(4)};
+  height: 16px;
+  padding-bottom: auto;
+  padding-left: 4px;
+  padding-right: 4px;
+  padding-top: auto;
```

Comment:
Hi @Kanav-Arora, thank you for contributing!

In the design system there are different sizes for the `Tag` component, which are not implemented yet: https://www.figma.com/file/xt8O9mFeLl46C5InWwoMrN/Twenty?type=design&node-id=3063%3A54900&mode=design&t=Oz0MjRJZUU0tq8P1-1

Could you implement a `size` prop for the `Tag` component, with possible values: `small`, `medium` and `large`, according to Figma? The default would be "large" as it is the closest to the existing one (height: 24px, padding-left and right: 8px. We can remove the padding top and bottom).
Then we would use `size="small"` in `SettingsObjectItemTableRow.tsx`.

Also, please use the `theme.spacing(...)` method in the styles (`theme.spacing(1)` is 4px, `theme.spacing(2)` is 8px, etc.).

Reply:
Sure
I'll do the needful